mm2src/coins/utxo/rpc_clients.rs

```diff
@@ -612,14 +612,15 @@ impl UtxoRpcClientOps for NativeClient {
     }
 
     fn display_balance(&self, address: Address, _decimals: u8) -> RpcRes<BigDecimal> {
-        Box::new(
-            self.list_unspent_impl(0, std::i32::MAX, vec![address.to_string()])
-                .map(|unspents| {
-                    unspents
-                        .iter()
-                        .fold(BigDecimal::from(0), |sum, unspent| sum + unspent.amount.to_decimal())
-                }),
-        )
+        let addresses = match address.addr_format {
+            UtxoAddressFormat::Segwit => vec![address.to_segwitaddress().unwrap().to_string()],
```

Comment:
Could you please replace this with `.expect("to_segwitaddress should not fail for UtxoAddressFormat::Segwit")`? I guess it's hard to avoid unwrap now, but in case of occasional panic, the reason will be a bit more clear with `expect()`.

Reply:
actually, I am currently adding a check here https://github.com/KomodoPlatform/atomicDEX-API/blob/b4f1aa169938f8985ef62c5abd51e22ba3ca81cd/mm2src/coins/utxo.rs#L1244 for bech32_hrp this way unwrap will not fail. But I will add `expect()` also for a more clear reason.